cmd/registry/cmd/config/configurations/create.go

```diff
@@ -38,8 +38,15 @@ func createCommand() *cobra.Command {
 				return fmt.Errorf("Cannot create config %q, it already exists.", name)
 			}
 
-			s := config.Configuration{}
-			err := s.Write(name)
+			// attempt to clone the active config or at least the current flags
+			s, err := config.Active()
```

Comment:
I struggled to understand this line, maybe because "Active" is an adjective and not a noun. That's the active configuration, right?

Reply:
Where are the flag values (like "registry.project") being saved?

Reply:
That is the active configuration, yes... making that `config.ActiveConfiguration()` would seem like a stutter to me. The flag values are automatically populated while reading a configuration using flag bindings. See: https://github.com/theganyo/registry/blob/main/pkg/config/config.go#L174